Common/Models/RuntimeConfiguration.cs

```diff
@@ -0,0 +1,9 @@
+namespace Common.Models
+{
+    public class RuntimeConfiguration
+    {
+        public const string SectionName = "RuntimeConfiguration";
+        public bool IsSqlite { get; set; }
+        public bool IsQueueAvailable { get; set; }
+    }
```

Comment:
1. We shouldn't disable external services for testing, since we're loosing ability to test calls to them. So instead of IsQueueAvailable flag:

- configure mock for queue in MockWebApplicationFactory.ConfigureWebHost method like it's done for other external services

2. Since IsSqlite is leaving alone and generally there is no need to configure anything that can be resolved at runtime, we can get rid of it by:

- adding a property like DatabaseContext.SupportsPredicates => Database.ProviderName != "Microsoft.EntityFrameworkCore.Sqlite"
- and passing this value to filter extensions instead of config

Reply:
Noted

Reply:
resolved it, have a nice holiday :)